modules/slo/main.tf

```diff
@@ -47,7 +47,17 @@ resource "datadog_service_level_objective" "monitor_slo" {
   force_delete = lookup(each.value, "force_delete", true)
   validate     = lookup(each.value, "validate", false)
 
-  tags = lookup(each.value, "tags", module.this.tags)
+  # convert terraform tags map to datadog tags map
+  # if a key is supplied with a value, it will render "key:value" as a tag
+  #   tags:
+  #     key = value
+  # if a key is supplied without a value (null), it will render "key" as a tag
+  #   tags:
+  #     key = null
+  tags = [
+    for tagk, tagv in lookup(each.value, "tags", module.this.tags) :
+    tagv != null ? format("%s:%s", tagk, tagv) : tagk
+  ]
```

Comment:
can this be moved to a local and copied for `datadog_service_level_objective.monitor_slo` and `datadog_service_level_objective.metric_slo`